requirements/base.in

```diff
@@ -14,3 +14,6 @@ Flask-Migrate
 
 # Deployment
 python-terraform
```

Comment:
Is this still needed?

Reply:
At the moment yes, unfortunately - our test still uses that via `Deployer.destroy` and `Deployer.refresh`. See #39.